Neos.NodeTypes.AssetList/composer.json

```diff
@@ -84,7 +86,14 @@
             "Neos.Neos-20161219122512",
             "Neos.Fusion-20161219130100",
             "Neos.Neos-20161220163741",
-            "Neos.Fusion-20170120013047"
+            "Neos.Fusion-20170120013047",
+            "Neos.Neos-20170115114620",
+            "Neos.Flow-20170125103800",
+            "Neos.Seo-20170127154600",
```

Comment:
This one should not be there, as Neos.Seo is a separate package

Reply:
Shouldn't matter much, apparently the migration didn't do anything...